### ISSUE TITLE
OSX 10.7.5: latest Build is broken. (libuv)

### ISSUE THREAD

**Issue by johncburnham** [urbit/archaeology#35]
After building:

johns-imac:urbit johnburnham$ bin/vere -c pierone
vere: urbit home is /Users/johnburnham/Desktop/urbit-master/urb
loom: mapped 1024MB
time: ~2013.9.29..06.38.42..72de
ames: on localhost, UDP 54117.
http: live on 8082
generating 2048-bit RSA pair...
saving passcode in /Users/johnburnham/.urbit/~habpem-tognul.txt
(for real security, write it down and delete the file...)
! /~zod/arvo/~2013.8.17..05.13.56..0492/clay/:<[487 5].[599 7]>
! /~zod/arvo/~2013.8.17..05.13.56..0492/clay/:<[487 9].[487 33]>
! /~zod/arvo/~2013.8.17..05.13.56..0492/clay/:<[487 15].[487 32]>
! exit
~fospex-hapnel-tamted-rammus--riddur-difpec-binfyl-nosnys/try=>

which just hangs.

Completely deleted my urbit directory and downloaded fresh from GH. Looks like a bug in libuv or in our makefile. make output attached.

Basically just lots of warnings of the form:

src/unix/uv-dtrace.h:42:42: warning: '$' in identifier [-Wdollar-in-identifier-extension]
extern void __dtrace_probe$uv$tick__start$v1$766f6964202a$696e74(const void *, int);

Details here: https://groups.google.com/forum/#!topic/urbit-dev/aFIf8DajwGQ

**Comment by johncburnham** [comment]
Correction: bug does not appear on OSX 10.8.4. Changed title to reflect that.

**Comment by cgyarvin** [comment]
I'm on 10.7.5 and I don't see it.

**Comment by cgyarvin** [comment]
What's interesting about this bug is that if you look at the location, line
487 in %clay, you see a card %deem which I thought I wasn't using, and
which grep can't seem to find me using either.  I don't think it has
anything to do with libuv etc.  I think it's ours... but a very strange one.

**Comment by johncburnham** [comment]
But during make, why all those $ in identifier compiler warnings? Those
don't appear when I build on OSX 10.8.4.

Regardless, running a software update and seeing if that changes anything.

**Comment by cgyarvin** [comment]
That's some libuv weirdness related to the warning set.  Of no importance
at all.

**Comment by cdcarter** [comment]
The warnings are unrelated, I am almost sure of. Newer libuv Makefiles suppress that specific warning on OSX.

**Comment by johncburnham** [comment]
Yup. Nothing to do with libuv. It's an improperly set $URBIT_HOME that does it.

**Comment by cgyarvin** [comment]
What was it set to?

**Comment by johncburnham** [comment]
It was URBIT_HOME=/Users/johnburnham/Desktop/urbit-master/urb

Should have been URBIT_HOME=/Users/johnburnham/urbit/urb

File here: https://groups.google.com/d/msg/urbit-dev/aFIf8DajwGQ/I6QrB49REc0J
